apps/checkout-app/backend/auth.ts

```diff
@@ -80,11 +81,42 @@ export const isAuthenticated = async (req: NextApiRequest) => {
   return await jwtVerifier.verify(token);
 };
 
-export const isAuthorized = async (req: NextApiRequest) => {
+export const hasPermissions = (
+  tokenData?: JwtPayload,
+  permissions?: PermissionEnum[]
+) => {
+  if (!permissions?.length) {
```

Comment:
I'm not sure I get this logic. The function is called `hasPermissions`, and we pass `permissions` to it, and then if `permissions` length === 0 we return `true` as `hasPermissions` equals `true`...?

Reply:
This function says if does it has all `permissions` in `user_permissions`. When length === 0 we can early return true;

Reply:
I've updated naming